src/components/Login/Login.js

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import "./login.css";
+import { login } from "../../redux/login/actions/login";
+
+const LoginPage = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const successOrErrorMessage = useSelector((state) => state.login);
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+  const passwordRegex =
+    /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]{8,}$/;
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  const validateForm = (event) => {
+    event.preventDefault();
+    const email = document.getElementById("email").value;
+    const password = document.getElementById("password").value;
+    const passwordValid = passwordRegex.test(password);
+    const emailValid = emailRegex.test(email);
+    setEmailError("");
+    setPasswordError("");
+    if (emailValid && passwordValid) {
+      let userLogin = {};
+      userLogin.email = email;
+      userLogin.password = password;
+      dispatch(login(userLogin));
```

Comment:
let userLogin = {};
      userLogin.email = email;
      userLogin.password = password;
      dispatch(login(userLogin));
      
improve this code @PuniaMohit